<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSE</class>
 <widget class="QDialog" name="frmSE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto Unidad 3</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

font-family: Microsoft JhengHei;
font-size: 14px;
background-color: white
}

QPushButton
{

background-color: rgb(23,161,165);
border-radius: 10px;
color: white;

}

QPushButton:hover
{
background-color: rgb(77,183,187);
border-radius: 10px;
color: white;
}

QPushButton:pressed
{
background-color: rgb(23,161,165);
border-radius: 10px;
color: white;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(117, 0, 175);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 0, 175);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnAceptar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Toma de Decisiones:</string>
   </property>
   <widget class="QRadioButton" name="rB_ID3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_AL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AL</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_KNN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>KNN</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_RNA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RNA </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_NB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>NB</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>PYTHON</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
